.github/workflows/phpunit-sqlite.yml

```diff
@@ -149,12 +149,6 @@ jobs:
         run: |
           composer run --list | grep '^  test:integration ' | wc -l | grep 1
 
-      - name: Enable debug logging
```

Comment:
Same here and in stable31

Reply:
Cherry-picked from master
And you already merged stable31?